### ISSUE TITLE
Time zone of series

### ISSUE THREAD

**Issue by gonzsanc** [cuemacro/findatapy#44]
Hello,

Really nice library, not easy to find something similar anywere.

I would like to ask how do we set or get the timezone relative to intraday data. I am checking the Dukascopy data.

Regards.

Gonzalo.

**Comment by saeedamen** [comment]
The data is downloaded in GMT I think, you can set it in Pandas. One way to check it's GMT is to look at price action around 8:30am ET on NFP days (there should be a massive spike in vol).

**Comment by gonzsanc** [comment]
Hi, thank your your answer.

Each data source depends on the timezone the data provider uses. Some use UTC, others UTC+3, some set the daily close to EST time no matter the timezone,etc..

The only way to know it is understanding the data provider specific settings.

Regarding, Pandas, the time zones can be modified if the source date field has timezone enabled, that it is not the case.

Therefore, the only way to id properly is to define the original timezone in the source data.

Regarding the volatility, it is not constant, you cannot assess confidently the timezone by checking the standard deviation, for instance, of the hourly charts trying to assess when the open hours are happening. Instead, what I personally do is taking a few samples of data and comparing these prices with a chart directly, like TradingView or whatever.

To summarize, unless the timezone is specified in your data source or referenced in a metadata file, the only way to do it properly is comparing your dataset with another dataset you know si synched to a specific timezone.

And my question was regarding if there was a way in this library to get that info. I understand tha the feature is not availabe at this moment.

Thank you, again, for you time and for this library.

Best,

Gonzalo.
